src/explore-education-statistics-common/src/services/util/__tests__/combineMeasuresWithDuplicateLocationCodes.test.ts

```diff
@@ -0,0 +1,583 @@
+import combineMeasuresWithDuplicateLocationCodes, {
+  slashSeparatedStringMergeStrategy,
+} from '@common/services/util/combineMeasuresWithDuplicateLocationCodes';
+import { LocationFilter } from '@common/modules/table-tool/types/filters';
+import { TableDataResult } from '@common/services/tableBuilderService';
+
+describe('combineMeasuresWithDuplicateLocationCodes', () => {
+  test('does not affect results that do not contain locations with duplicate levels and codes', () => {
+    const tableDataResult: TableDataResult[] = [
+      {
+        filters: ['filter-1', 'filter-2'],
+        geographicLevel: 'provider',
+        timePeriod: '',
+        measures: {
+          'indicator-1': '10',
+          'indicator-2': '20',
+          'indicator-3': '30',
+        },
+        location: {
+          provider: {
+            code: 'provider-1',
+            name: 'Provider 1',
+          },
+        },
+      },
+      {
+        filters: ['filter-1', 'filter-2'],
+        geographicLevel: 'provider',
+        timePeriod: '',
+        measures: {
+          'indicator-1': '40',
+          'indicator-2': '50',
+          'indicator-3': '60',
+        },
+        location: {
+          provider: {
+            code: 'provider-2',
+            name: 'Provider 2',
+          },
+        },
+      },
+      {
+        filters: ['filter-1', 'filter-2'],
+        geographicLevel: 'localAuthority',
+        timePeriod: '',
+        measures: {
+          'indicator-1': '70',
+          'indicator-2': '80',
+          'indicator-3': '90',
+        },
+        location: {
+          localAuthority: {
+            code: 'localAuthority-1',
+            name: 'LocalAuthority 1',
+          },
+        },
+      },
+    ];
+
+    const deduplicatedLocations: LocationFilter[] = [];
+
+    const results = combineMeasuresWithDuplicateLocationCodes(
+      tableDataResult,
+      deduplicatedLocations,
+      slashSeparatedStringMergeStrategy,
```

Comment:
This strategy is only used in the tests?

Reply:
:+1: